tests/unit/einstein/test_ceph.py

```diff
@@ -0,0 +1,272 @@
+        removeSnapFlag = self.fs.remove_snapshot(CEPH_IMG, CEPH_SNAP_IMG)
+        self.assertTrue(removeSnapFlag, "Failed to remove a snapshot")
+        # remove parent image, note here remove order does not matter
+        removeImg = self.fs.remove(CEPH_IMG)
+        self.assertTrue(removeImg, "Failed to remove test parent image!")
+        # remove child image
+        removeChild = self.fs.remove(CEPH_CHILD_IMG)
+        self.assertTrue(removeChild, "Failed to remove test child image clone \
+                                       from parent image %s!" % CEPH_IMG)
+
+
+class TestWrite(unittest.TestCase):
+    """
+    Write image
+    """
+    @trace
+    def setUp(self):
+        self.fs = Ceph.RBD(_cfg.fs, _cfg.iscsi.password)
+        self.fs.create_image(CEPH_IMG, CEPH_IMG_SIZE)
+        time.sleep(HIL_CALL_TIMEOUT)
+
+    def runTest(self):
+        bytes_written = self.fs.write(CEPH_IMG, TEST_DATA, OFFSET)
+        data_read = self.fs.read(CEPH_IMG, OFFSET, bytes_written)
+        self.assertEqual(TEST_DATA, data_read)
+
+    def tearDown(self):
+        self.fs.remove(CEPH_IMG)
+        time.sleep(HIL_CALL_TIMEOUT)
+
+
+class TestSnapImage(unittest.TestCase):
+    """
+    create snapshot
+    """
+    @trace
+    def setUp(self):
+        self.fs = Ceph.RBD(_cfg.fs, _cfg.iscsi.password)
+        imgs = self.fs.list_images()
+        self.fs.create_image(CEPH_IMG, CEPH_IMG_SIZE)
+        time.sleep(HIL_CALL_TIMEOUT)
+
+    def runTest(self):
+        snapFlag = self.fs.snap_image(CEPH_IMG, CEPH_SNAP_IMG)
+        self.assertTrue(snapFlag)
+        snapList = self.fs.list_snapshots(CEPH_IMG)
+        self.assertIn(CEPH_SNAP_IMG, snapList)
+        self.assertNotIn(NOT_EXIST_SNAP_NAME, snapList)
+
+    def tearDown(self):
+        self.fs.remove_snapshot(CEPH_IMG, CEPH_SNAP_IMG)
+        self.fs.remove(CEPH_IMG)
+
+
+@unittest.skip("removeSnapShot class skipping")
+class TestListSnapshot(unittest.TestCase):
+    """
+    List Snapshot
+    """
+    @unittest.skip("removeSnapShot class skipping")
+    def setUp(self):
+        pass
+
+    def runTest(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+
+@unittest.skip("removeSnapShot class skipping")
+class TestRemoveSnapshot(unittest.TestCase):
+    """
+    Remove snapshot
+    """
+    @trace
+    def setUp(self):
+        pass
+
+    def runTest(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+
+class TestGetImage(unittest.TestCase):
+    """
+    Get image
+    """
+    @trace
+    def setUp(self):
+        self.fs = Ceph.RBD(_cfg.fs, _cfg.iscsi.password)
+        time.sleep(HIL_CALL_TIMEOUT)
+
+    def runTest(self):
+        good_img = self.fs.get_image(EXIST_IMG_NAME)
+        self.assertEqual(good_img.name, EXIST_IMG_NAME)
+        with self.assertRaises(file_system_exceptions.ImageNotFoundException):
+            bad_img = self.fs.get_image(NOT_EXIST_IMG_NAME)
+
+    def tearDown(self):
+        pass
+
+
+class TestMap(unittest.TestCase):
```

Comment:
Save this test in your local machine, but remove it from here as Map will be removed